.github/workflows/performance.yml

```diff
@@ -35,11 +35,11 @@ jobs:
         run: |
           ./build_aws_sdk_unix.sh $BUILD_TYPE
 
-      - name: 'Set up JDK 8'
+      - name: 'Set up JDK'
         uses: actions/setup-java@v4
         with:
           distribution: 'corretto'
-          java-version: 8
+          java-version: 17
```

Comment:
Curious why are we bumping the Java version to 17?

Reply:
Upgrading the Gradle version requires upgrading to a compatible JDK version. And these upgrades match the Gradle and JDK versions used by `aws-pgsql-odbc`.